packages/debug-test/package.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "debug-test",
+	"version": "0.0.0",
+	"private": "true",
+	"scripts": {
+		"test": "jest"
+	},
+	"devDependencies": {
+		"@react-native-windows/automation": "^0.1.73",
+		"@types/jest": "^26.0.20",
+		"ts-jest": "^26.1.4"
+	}
```

Comment:
This should be including the shared scripts/configuration used by other packages/tests in the repo. E.g. the default `rnw-scripts lint` task, and `rnw-scripts test` task, that standardize configuration.

Other TS tests also avoid `ts-jest` usage in favor of using `babel` support; That configuration should be mostly out of the box if you use `rnw-scripts test` and include the correct dependencies.

Reply:
I added "@rnw-scripts/eslint-config" to the devDependencies. Hope that's what you meant - if not, could you be more specific? I'm not familiar with the other things you mention. Are they entries in the package.json file? CLI commands?

Reply:
The repo is set up as separately isolated packages, akin to something like liblets. Each package has its own "tasks", which are akin to "targets". E.g. build, clean. One of these tasks, that is present in the other JS packages in the repo is "lint". This runs static analysis rules  (akin to oacr), type checking, format checking, etc.

So, adding a "lint" command to the package.json is similar to adding a new project to "oacr.projects.ini".

Each project has a separate ".eslintrc.js" file, which allows configurating how the project is linted. Most of the time, this uses a shared configuration, which includes all of the checks, we want to run across our entire repo.

The package.json of "@rnw-scripts/just-task" has a field under "bin", requesting that the command "rnw-scripts" invokes an entry point in that package. So that is what "rnw-scripts build", "rnw-scripts lint" etc will invoke. 

@dannyvv has expertise here as well, and could help add on anything I'm missing.